enterprise-suite/es-grafana/akka-stream-exporter.json

```diff
@@ -0,0 +1,50 @@
+[
+    {
+        "graphName": "Stream Throughput",
+        "promQL": [
+            {
+                "expr": "rate(akka_stream_stream_throughput{ContextTags}[1m])",
+                "legendFormat": "{{stream}}"
+            }
+        ],
+        "yaxes": {
+            "format": "ops"
+        }
+    },
+    {
+        "graphName": "Stream Processing Time",
+        "promQL": [
+            {
+                "expr": "akka_stream_stream_flow_time_ns{ContextTags}",
+                "legendFormat": "{{stream}} quantile {{quantile}}"
+            }
+        ],
+        "yaxes": {
+            "format": "ns"
+        }
+    },
+    {
+        "graphName": "Slowest Operators",
```

Comment:
Should we document here this is the 99th percentile?

Reply:
I was hoping "slowest" was a good explanation. Maybe put quantile in the legend?